kazoo/testing/sslfixtures.py

```diff
@@ -0,0 +1,138 @@
+ca_cert = """
```

Comment:
Would it be possible to generate all those certificates in a dynamic way (using pyOpenSSL for instance)?